k3/k3r/k3r.go

```diff
@@ -111,6 +111,7 @@ func (t *Test) RunWithContext(ctx context.Context) <-chan tests.Event {
 			cmd.Args = append(cmd.Args, strings.Fields(s)...)
 		}
 		cmd.Dir = t.Dir
+		cmd.Env = append(cmd.Env, os.Environ()...)
```

Comment:
I my experience it is better for a library like this not to add the os environment, but to let the user decide these things.

If you want to fix the unit tests for our k3 ci, maybe add os.Environ in the unit test instead?

Reply:
it is essential to pass the environment (or at least parts of it) to the child process. In the end only the author of the child process executable will know what environment is suitable, or we as the caller. Having the environment passed only as part of the test seems not to be an adequate solution here

Reply:
What parts of the environment are required by k3r? Adding them explicitly seems like a good option.